File: FizzBuzz/fizzbuzz.js
class FizzBuzz {
  fizzBuzz(start, end) {
    let ar = [];
    let index = start;

    while (index <= end) {
      if (index % 3 === 0 && index % 5 === 0) {
        ar.push("FizzBuzz");
      } else if (index % 3 === 0) {
        ar.push("Fizz");
      } else if (index % 5 === 0) {
        ar.push("Buzz");
      } else {
        ar.push(String(index));
      }

      index++;
    }

    return ar;
  }
}

// Example usage
const fizzBuzzGame = new FizzBuzz();
const results = fizzBuzzGame.fizzBuzz(1, 100);
console.log(results);

File: FizzBuzz/fizzbuzz.py
class FizzBuzz:
    def __init__(self):
        pass
    
    def fizz_buzz(self, start, end):
        ar = []
        
        index = start
        
        while index <= end:
            if index % 3 == 0 and index % 5 == 0:
                ar.append("FizzBuzz")
            elif index % 3 == 0:
                ar.append("Fizz")
            elif index % 5 == 0:
                ar.append("Buzz")
            else:
                ar.append(str(index))
            
            index += 1
        
        return ar

# Example usage
fizz_buzz_game = FizzBuzz()
results = fizz_buzz_game.fizz_buzz(1, 100)
print(results)
